common/src/main/java/com/wynntils/features/chat/ChatItemFeature.java

```diff
@@ -145,8 +151,26 @@ public void onChatReceived(ChatMessageReceivedEvent e) {
         e.setMessage(modified.getComponent());
     }
 
-    private void onInventoryPress(Slot slot) {
-        makeChatPrompt(slot);
+    private void onInventoryPress(Slot hoveredSlot) {
+        if (hoveredSlot == null) return;
+
+        // Special case for unidentified gear
+        Optional<GearItem> gearItemOpt = Models.Item.asWynnItem(hoveredSlot.getItem(), GearItem.class);
+        if (gearItemOpt.isPresent() && gearItemOpt.get().isUnidentified()) {
+            // We can only send chat encoded gear of identified gear
+            WynntilsMod.warn("Cannot make chat link of unidentified gear");
```

Comment:
Why's that, though..? I mean, wouldn't it be nice if we could send unidentified gear as well? Sure, you could type the name and the recipient can look it up in the item DB, but sending it as a chat link would be much easier. Maybe something to consider for the new chat protocol, @kristofbolyai ?